assets/pages/new-contributions-landing/components/StripePaymentRequestButton.jsx

```diff
@@ -160,7 +160,7 @@ function initialisePaymentRequest(props: PropTypes) {
     // We need to do this so that we can offer marketing permissions on the thank you page
     updateUserEmail(data, props.updateEmail);
     const tokenId = props.isTestUser ? 'tok_visa' : token.id;
-    props.onPaymentAuthorised({ paymentMethod: 'Stripe', token: tokenId })
+    props.onPaymentAuthorised({ paymentMethod: 'Stripe', token: tokenId, stripePaymentMethod: 'StripeApplePay' })
```

Comment:
do we need to double up on the "stripe" part, the "stripe payment method" is just apple pay.  Also stripe is the payment method, we're overloading the word payment method.  We could have to change to payment processor and payment experience or something.

Reply:
It will end up as `StripeApplePay` in the `PaymentProvider` field in the datalake. It is overloading the field, but we decided that there wasn't enough to gain by splitting out the two parts of the payment